### DISCUSSION TITLE
account recovery when user is in several organizations

### DISCUSSION THREAD

**Discussion by leclou69** [dani-garcia/vaultwarden#4960] [Q&A] [answered]
Hi,
I've read everywhere that it is necessary that a user is member of only one unique organization for account recovery to work.
Just tested it in a lab environment with a user who is member of two organizations, in both his account is registered for account recovery.
After changing / recovering his password, nothing weitd happens to his account.
He can still access each item as before.
So, is it really necessary to activate the policy for unique organizations?
Using version 1.32.0
Thanks in advance

**Comment by stefan0xC** [comment, marked as answer]
It's not necessary but it might be a security concern for some (if a user is part of multiple organizations you might not want another organization's admin to be able to get access via the password reset functionality of your member). So the option to enforce this was enabled recently by #4903 (available in the `testing` image).
https://github.com/dani-garcia/vaultwarden/blob/6ceed9284d16ba11cae4515d6ab666225725cb99/.env.template#L428-L432
